python/paddle/fluid/tests/unittests/test_mul_ngraph_op.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
How about create a separate directory `python/paddle/fluid/tests/unittests/ngraph/` to put all ngraph unittests?

Reply:
The configuration of test was prepared here: https://github.com/PaddlePaddle/Paddle/pull/14800